### PR TITLE
Fix the buttons for screen sizes from 426px - 670px Issue #212

### LINKED ISSUE #212: Fix the buttons for screen sizes from 426px - 670px and reduce the font size of heading
image

- In the above image, we can that the buttons are not of same dimensions. The content in it won't allow 3 buttons in a row for smaller screen size (426px- 670px). Hence, we need to change the layout of buttons from row --> column. Please refer the below image for the same.

image

- The font size of the text `Check Out Our Docs` looks a bit large. Please change it to `1.8rem` for screens between `426px - 670px`. Refer the below image.
image

### PR DESCRIPTION
**Pull request by oneonone97**
Closes #212

### CODE REVIEW

**Review by Ishikabaid — CHANGES_REQUESTED** [review]
ss
You have changed the buttons for all screen sizes above 426, weren't you supposed to change them for `426px-670px` only?

ss1
Also please refer to the design for `426px-670px` , the width of the buttons should be more and there should be margin between them.

File: css/index.css [hunk 1]
```diff
@@ -310,12 +317,22 @@ footer {
...
+  .docs-heading {
+    font-size: 1.8rem;
+  }
+
   .cards {
-    width: 85%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
```
Comment (Rucha1499) [inline css/index.css:328]: image
The content the last card/button looks a bit crowded. Can you fix this please? Either the card size can be increased or font-size can be reduced. Please check the breakpoints and make the changes accordingly!

**Review by Rucha1499 — CHANGES_REQUESTED** [review]
Thanks for the PR @oneonone97 🎉 
Adding few comments. Kindly check!

Thank you!

File: css/index.css (hunk 1 above)
Comment (oneonone97) [reply css/index.css:328]: Edited check it out now!
Thank You 😊

**Review by Rucha1499 — APPROVED** [review]
Looks good to me!

File: css/index.css [hunk 2]
```diff
@@ -278,6 +278,13 @@ footer {
     font-size: 0.9rem;
   }
 
+  .card {
+    padding: 15px 1px;
+    width: 150%;
+    font-size: 1rem;
```
Comment (rohan09-raj) [inline css/index.css:284]: By default the font size is 1 rem, you don't need to explicitly assign it until you have to override the font-size

File: css/index.css [hunk 3]
```diff
@@ -278,6 +278,13 @@ footer {
     font-size: 0.9rem;
   }
 
+  .card {
+    padding: 15px 1px;
```
Comment (rohan09-raj) [inline css/index.css:282]: Can we make it 2px ?? 1px is a bit negligible

File: package-lock.json [hunk 4]
```diff
@@ -1,222 +1,8 @@
...
-        "prettier": "2.2.1"
-      }
-    },
-    "node_modules/buffer-from": {
-      "version": "1.1.1",
-      "resolved": "https://registry.npmjs.org/buffer-from/-/buffer-from-1.1.1.tgz",
-      "integrity": "sha512-MQcXEUbCKtEo7bhqEs6560Hyd4XaovZlO/k9V3hjVUF/zwW7KBVdSK4gIt/bzwS9MbR5qob+F5jusZsb0YQK2A==",
-      "dev": true
-    },
-    "node_modules/concat-stream": {
```
Comment (rohan09-raj) [inline package-lock.json:22]: Have you not done npm install while setting up the project ??

File: css/index.css (hunk 2 above)
Comment (oneonone97) [reply css/index.css:284]: I tried it but the button overflow(last button)

File: css/index.css (hunk 3 above)
Comment (oneonone97) [reply css/index.css:282]: For this it's not responsive

File: package-lock.json (hunk 4 above)
Comment (oneonone97) [reply package-lock.json:22]: I'm using node version `14.x` and I can go with higher version of node but for using `yarn` the Node version should be `14.x`

**Review by rohan09-raj — APPROVED** [review]
Good work @oneonone97 🥳 💯, approved from my side

**Pull request merged**